app/sample_manifest_excel/sample_manifest_excel/upload/row.rb

```diff
@@ -43,7 +43,11 @@ def at(col_num)
       ##
       # Find a value based on a column name
       def value(key)
-        at(columns.find_column_or_null(:name, key).number)
+        column_number = columns.find_column_or_null(:name, key).number
```

Comment:
better way to write this check? null.number may be returning -1 and we don't want that